src/Bijectors.jl

```diff
@@ -249,6 +252,11 @@ include("utils.jl")
 include("interface.jl")
 include("chainrules.jl")
 
+Base.@deprecate forward(b::AbstractBijector, x) with_logabsdet_jacobian(b, x)
```

Comment:
AFAICT `forward` returns a NamedTuple whereas `with_logabsdet_jacobian` returns a Tuple? So this deprecation seems to be incorrect (and hence introduce an unintended breaking change - otherwise one could just remove `forward` directly). The following should work:
```suggestion
Base.@deprecate forward(b::AbstractBijector, x) NamedTuple{(:rv,:logabsdetjac)}(with_logabsdet_jacobian(b, x))
```

Reply:
Oh, neat!